assets/api.ts

```diff
@@ -0,0 +1,24 @@
+import {IArticle} from './interfaces';
+import {gettext} from './utils';
+
+export interface INewshubApi {
+    ui: {
+        wire: {
+            getVersionsLabelText(item: IArticle, plural?: boolean): string;
+        };
+    };
+    localization: {
+        gettext(message: string, params?: {[key: string]: string | number}): string;
+    };
+}
+
+function getVersionsLabelText(_item: IArticle, plural?: boolean): string {
```

Comment:
I'm wondering if we should have rather the function return some predefined options like:
```ts
getAncestorType(_item: IArticle): 'versions' | 'sequence';
```
and handle those options in core, that would make translations etc simpler

Reply:
@petr to keep it simple for now, I'll move this code into core. We can move it out later if desired.